packages/xod-arduino/src/transpiler.js

```diff
@@ -401,3 +407,7 @@ export const transformProjectWithDebug = def(
 );
 
 export const transpile = renderProject;
+
+export const forUnitTests = {
```

Comment:
A new pattern of exporting private functions for tests?
Will do the same thing everywhere in the future?

Reply:
@brusherru @evgenykochetkov Just a proposal.

I find it useful because it’s easier to scan a module to understand what is its “real” API. Often the API includes only a tiny fraction of all symbols of the global scope. Having the explicit split one could be more focused when reviewing essential public API functions and relax a bit when looking at very local implementation details.

If you agree, let’s use it.